public/script.js

```diff
@@ -2608,14 +2617,17 @@ class StreamingProcessor {
         try {
             const sw = new Stopwatch(1000 / power_user.streaming_fps);
             const timestamps = [];
-            for await (const { text, swipes } of this.generator()) {
+            for await (const { text, swipes, logprobs } of this.generator()) {
```

Comment:
Streaming response async generator now returns `logprobs` on supported APIs, but any code that uses this value is tolerant of null logprobs so this will not affect other APIs that don't have logprob support implemented.